calendar/calendar-api/api/src/java/org/sakaiproject/calendar/api/CalendarConstants.java

```diff
@@ -19,4 +19,5 @@ public class CalendarConstants {
 
 	public final static String NEW_ASSIGNMENT_DUEDATE_CALENDAR_ASSIGNMENT_ID = "new_assignment_duedate_calendar_assignment_id";
 	public final static String NEW_ASSIGNMENT_OPEN_DATE_ANNOUNCED = "new_assignment_open_date_announced";
+	public final static String TOOL_ID = "calendar_event_tool_id";
```

Comment:
I would definitely be using sakai.calendar here, and update site-manage to use this constant, too.

Reply:
Ah - You misunderstand the purpose of this variable - I will rename.  Thanks.